### ISSUE TITLE
Unable to use common filters in custom templates

### ISSUE THREAD

**Issue by willeccles** [OpenCyphal/nunavut#310]
(Copied from original forum post [here](https://forum.opencyphal.org/t/unable-to-use-common-filters-in-custom-templates/1910/1).)

I intended to write my own templates for C generation, so I went ahead and copied the contents of `lang/c/templates` to a new directory. Before attempting to modify them, I tried to generate C headers with these new templates. The command I am running is as follows (I am testing with the `public_regulated_data_types`):

```
nnvg --target-language c --target-endianness=little --enable-serialization-asserts --templates templates public_regulated_data_types/reg --lookup-dir public_regulated_data_types/uavcan
```

If I simply remove `--templates`, all is well and it generates normally. However, after adding the template directory, I get the following:

```
nunavut.jinja.jinja2.exceptions.TemplateAssertionError: no filter named 'remove_blank_lines'
```

What am I missing? I can't seem to make this work at all. The documentation suggests that I should be able to use these just fine.

**Comment by thirtytwobits** [comment]
Huh. I have this use case covered in the test suites. Let me take peek...

**Comment by thirtytwobits** [comment]
Confirmed. Looking into the issue now...

**Comment by thirtytwobits** [comment]
fyi, the bug is in the serialization code generator. If you use `--omit-serialization-support` it does work.

**Comment by thirtytwobits** [comment]
Found one bug related: support loaders shouldn't use the external template path as "support" == "internal" in Nunavut. Because of this the loaders are getting confused about duplicate `serialization.j2` templates.

**Comment by willeccles** [comment]
> fyi, the bug is in the serialization code generator. If you use `--omit-serialization-support` it does work.

Confirmed, this fixes it. This is actually excellent news for my use case. I need to use the default templates once, then use my templates for some auxiliary things (which do not include serialization). Thanks for looking into this!

**Comment by willeccles** [comment]
Closing as this is fixed by #311 (2.1.1).
